### PR TITLE
BA-243-iDEAL-Processing-not-working-properly-therefore-create-a-separate-method-for-iDEAL-Processing

### PR DESCRIPTION
**Pull request by vegimcarkaxhija**
add Ideal Processing as separate payment method, also removed duplicte NoServiceSpecifiedPayment in $payments array

### CODE REVIEW

**Review by SandervdHulst — APPROVED** [review]

File: src/PaymentMethods/PaymentMethodFactory.php [hunk 1]
```diff
@@ -120,7 +122,6 @@ class PaymentMethodFactory
         Przelewy24::class => ['przelewy24'],
         PointOfSale::class => ['pospayment'],
         Giropay::class => ['giropay'],
-        NoServiceSpecifiedPayment::class => ['noservice'],
```
Comment (ShuCh3n) [inline src/PaymentMethods/PaymentMethodFactory.php:123]: why is this removed?

Comment (vegimcarkaxhija) [reply src/PaymentMethods/PaymentMethodFactory.php:123]: it was writen twice, you can check line 120 is still there
